src/lib/Addresses.svelte

```diff
@@ -1,5 +1,8 @@
+<script context="module" lang="ts">
+    export const config = { renToken: "", renPool: "" };
```

Comment:
not sure if this component should depend on the context, maybe we could just remove the context and expose a prop called `addresses`: 
```
export let addresses = {[key: string]: string}
```
Then, from the parent component you'd pass:
```
addresses = {
  owner: '0x',
  nodeOperator: '0x',
  renToken: '0x',
  renPool: '0x',
}
```

Reply:
Yeap, we can do that.